src/app/subnets/views/SubnetDetails/SubnetStaticIPs/StaticDHCPTable/StaticDHCPTable.test.tsx

```diff
@@ -0,0 +1,14 @@
+import StaticDHCPTable from "./StaticDHCPTable";
+
+import { renderWithBrowserRouter, screen } from "@/testing/utils";
+
+it("renders a static DHCP table", () => {
+  renderWithBrowserRouter(<StaticDHCPTable staticDHCPLeases={[]} />);
```

Comment:
Please add a test for a scenario where there are rows in the table. This will require you adding a new factory (similar to the ones in `src/testing/factories/subnet.ts`.